torchvision/transforms/functional.py

```diff
@@ -200,15 +196,21 @@ def normalize(tensor, mean, std, inplace=False):
     Returns:
         Tensor: Normalized Tensor image.
     """
-    if not _is_tensor_image(tensor):
-        raise TypeError('tensor is not a torch image.')
+    if not torch.is_tensor(tensor):
+        raise TypeError('tensor should be a torch tensor. Got {}.'.format(type(tensor)))
+    
+    if tensor.ndimension() != 3:
+        raise ValueError('Expected tensor to be a tensor image of size (C, H, W). Got tensor.size() = '
+                         '{}.'.format(tensor.size()))
 
     if not inplace:
         tensor = tensor.clone()
 
     dtype = tensor.dtype
     mean = torch.as_tensor(mean, dtype=dtype, device=tensor.device)
     std = torch.as_tensor(std, dtype=dtype, device=tensor.device)
+    if std == 0:
```

Comment:
This doesn't work if `std` is not a 1-element tensor (like if we pass `std=[1, 1, 1]` for example).
Could you do instead
```python
if (std == 0).any():
    ...
```
This is why tests are failing with
```
RuntimeError: bool value of Tensor with more than one value is ambiguous
```
Could you fix this?

Reply:
Done, thank you!